modules/shared/partials/network-requirements.adoc

```diff
@@ -8,10 +8,10 @@ For this reason, only LAN-like network environments are officially supported.
 Couchbase does document, for purposes of convenience when developing and performing basic operational work, what may need to be tuned when network throughputs and latencies are higher.
 If you encounter issues, even with these tune-ables, you should attempt the same workload from a supported, LAN-like environment.
 
-[IMPORTANT]
-.Running on AWS Lambda
-====
-AWS Lambda's execution environment can freeze/thaw processes. Current Couchbase SDKs run background processing to check and adapt to topology changes in the Couchbase Cluster being used. 
-Since the freeze does not allow this background processing and upon thaw the event-driven Couchbase SDKs may issue requests to an old topology, unexpected behavior including failures may occur. 
-AWS λ does not provide a way to detect the freeze/thaw cycle. For this reason AWS λ is not currently a tested and supported platform for running current Couchbase SDKs.
-====
+=== Serverless Environments
+
+SDK API 3.4 introduces better resilience in handling errors that may occur when running your application in serverless environments, in particular when processes are frozen or thawed and a rebalance is required.
+This means official support for AWS Lambda, Azure Functions and GCP Functions.
+
+NOTE: When *DNS SRV* records are used to connect to the SDK it is possible for the underlying addresses to change (i.e. the cluster could move).
+The SDK will detect this and react accordingly so that your application can continue to work correctly.
```

Comment:
@programmatix I recall some discussion about how there could be issues with the "Transactions background process" with Serverless Environments, is there anything we should add here for users to be aware of?